socorro/processor/cache_manager.py

```diff
@@ -217,11 +220,21 @@ def make_room(self, size):
             total_size -= rm_size
             removed += rm_size
             try:
+                # Delete the evicted file
                 os.remove(rm_path)
             except FileNotFoundError:
                 # If there was an OSError, then this file is gone already. We need to
                 # update our bookkeping, so continue.
                 pass
+            try:
+                # Attempt to prune empty directories. This will trigger DELETE | ISDIR
+                # events and get cleaned up by the event loop.
+                os.removedirs(os.path.dirname(rm_path))
+            except OSError as exc:
+                if self.is_verbose:
+                    self.logger.debug("unable to remove directory: %s %s", rm_path, exc)
```

Comment:
I can take this out. It's almost assuredly because there's stuff in the directory.